README.md

```diff
@@ -4,5 +4,24 @@ _Extracts articles from [Wikipedia database dumps](https://en.wikipedia.org/wiki
 
 ## Usage
 
+To serve as a drop-in replacement for the descriptions scraper:
+- Install this tool to `$PATH` as `om-wikiparser`.
+- Download [the dumps in the desired languages](https://dumps.wikimedia.org/other/enterprise_html/runs/) (Use the files with the format `${LANG}wiki-NS0-${DATE}-ENTERPRISE-HTML.json.tar.gz`).
+- Set `WIKIPEDIA_ENTERPRISE_DUMPS` to the list of the dump files to process
+- Run the following from within the `intermediate_data` subdirectory of the maps build directory:
+```shell
+# transform intermediate files from generator
```

Comment:
Pros:
- Independent of the generator process. Can be run as soon as planet file is updated.
Cons:
- Need to keep osm query in sync with generator's own multi-step filtering and transformation process.
- Need to match generator's multi-step processing of urls exactly.

When I did this earlier, it was with the `osm-filter` tool, I only tested it on the yukon region, and it output _more_ entries than the generator did.

I can create an issue for this, but the rough steps to get that working are:
- Convert `osmfilter` query to `osmium` command so it can work on `pbf` files directly.
- Dig into generator map processing to try to improve querying.
- Compare processing of a complete planet with generator output.
- Write conversion of `osmuim` output for `wikiparser` to use.

Reply:
1. Does it make sense to create an issue to document the existing generator's output format, and propose some improvements if necessary? What kind of complex transformations are done in the generator now, and why?
2. What's wrong in outputting more URLs? I assume that the generator may filter now OSM POIs/types that we are not supporting yet. In the worst case, some more articles will be extracted from the planet, right? Do you remember how big is the percent of "unnecessary" articles?
3. osmfilter can work with o5m, osmconvert can process pbf. There is also https://docs.rs/osmpbf/latest/osmpbf/ for direct pbf processing if it makes the approach simpler. How good is the osmium tool compared to other options?

It would be great to have a well-defined and independent API between the generator and wikiparser, to avoid complications when supporting it in the longer term. WDYT?

Reply:
> I think so, do you mean the wikipedia/wikidata files or the mwm format in general?

I meant those files that are required for wikiparser to work. It actually may make sense to keep it in README or some other doc, not in an issue.